core/Node.js

```diff
@@ -922,8 +922,8 @@ Node.prototype._vecOptionalSet = function _vecOptionalSet (vec, index, val) {
  * @return {Node} this
  */
 Node.prototype.show = function show () {
-    Dispatch.show(this.getLocation());
     this._shown = true;
```

Comment:
Ideally `this._shown` shouldn't be set here at all. Instead `node._setShown(true);` should be called in `Dispatch#show`.